README.md

```diff
@@ -104,24 +104,25 @@ Alternatively, you can also run the OSS Review Toolkit by building its Docker im
 The Analyzer determines the dependencies of software projects inside the specified input directory (`-i`). It does so by
 querying whatever [supported package manager](./analyzer/src/main/kotlin/managers) is found. No modifications to your
 existing project source code, or especially to the build system, are necessary for that to work. The tree of transitive
-dependencies per project is written out as [ABCD](https://github.com/nexB/aboutcode/tree/master/aboutcode-data)-style
-YAML (or JSON, see `-f`) file named `analyzer-result.yml` to the specified output directory (`-o`). The output file
+dependencies per project is written out as part of an
```

Comment:
commit message nit: type in headline, e.g. ACBD

Reply:
Done.